components/favicon_title_handler/favicon_title_handler.tsx

```diff
@@ -83,21 +84,32 @@ class FaviconTitleHandler extends React.PureComponent<Props> {
             currentTeam,
             currentTeammate,
             unreads,
+            inGlobalThreads,
         } = this.props;
         const {formatMessage} = this.props.intl;
 
         const currentSiteName = siteName || '';
 
+        const mentionTitle = unreads.mentionCount > 0 ? '(' + unreads.mentionCount + ') ' : '';
```

Comment:
I was tempted to make aside improvements like this, but this was how it was—it may have shifted, but I didn't want to change anything unnecessarily.

Reply:
> I used `/test threaded_post` and it seems it takes a while to load, but then I can scroll with no delay. I'm not really sure how that would work with really long threads in customer setups.
> Also, while waiting for the thread to load I didn't have any sort of loading animation, but I guess this could be in a future PR.

@Willyfrog Unfortunately this is about what we have now—opening long threads in RHS will do a blocking render until all posts have loaded+rendered.
<img width="179" alt="CleanShot 2021-04-06 at 22 03 49@2x" src="https://user-images.githubusercontent.com/11724372/113804349-fc735b00-9723-11eb-82b6-4f5e279438cc.png">
But I agree, we should make some improvements (ThreadViewer infinite scrolling with non-blocking rendering, loading indicators, etc.)
 
 
> Also, by creating a thread with that command, it didn't collapse into a single message

Full support for the in-channel experience will come in subsequent PRs. Some server/mm-redux work that Eli did got us part way there with Init channel-loading only including root posts, but a follow up PR is needed to explicitly disregard replies (right now they come in via WS events), add ThreadFooter, and a number of other items.

> some more weird stuff. I created a second account and joined the `guille` team. this new user didn't see any replies, but just the initial message (screenshot #1). After that I clicked the reply button and closed the RHS, and now it showed that there were 7 replies (screenshot #2). Finally I replied and my global threads lighted as if there were messages unread, but the original user didn't get anything (screenshot #3)

I'm hoping all of this is related to a DB migration issue. I re-created this test server, so replies in Global Threads should be working as-expected, though as previously mentioned: sidebar mentions/unread indicators are still in-progress until #7733 is merged.